### DISCUSSION TITLE
请增加arm_cortex-a7_neon-vfpv4架构编译，谢谢大佬！

### DISCUSSION THREAD

**Discussion by dlonzyz** [messense/aliyundrive-webdav#131] [Q&A] [answered]
arm_cortex-a7_neon-vfpv4，大佬辛苦！

**Comment by messense** [comment]
什么路由器？

**Reply by dlonzyz** [reply]
竟斗云2.0 / G-DOCK 2.0 (IPQ40xx R619AC)

**Reply by dlonzyz** [reply]
<html>
<body>


P&W R619AC 128M (CpuMark : 8090.205795 Scores)
--
架构 | ARMv7 Processor rev 5 (v7l) x 4


</body>
</html>

**Comment by messense** [comment, marked as answer]
https://github.com/messense/aliyundrive-webdav/releases/download/v0.5.4/aliyundrive-webdav_0.5.4_arm_cortex-a7_neon-vfpv4.ipk
